tests/src/test/scala/whisk/core/cli/test/ApiGwRestTests.scala

```diff
@@ -245,4 +248,48 @@ class ApiGwRestTests extends ApiGwTests {
     rr.stderr should include("A valid auth key is required")
   }
 
+  behavior of "Wsk rest api creation with path parameters with swagger"
```

Comment:
@jessealva Since path parameter testing is common to RESTful and CLI based testing, this test may be defined in ApiGwTests.scala and automatically carried forward into the REST and CLI extensions of th ApiGwTests class.

Reply:
So it kinda can, but id have to add logic to tell whether the output is JSON (from REST) or 2 lines of text (from CLI).  It was cleaner just to put it here as the logic is tied to the JSON thats returned when doing a create, which we then have to parse to validate.  This output is completely different when running via the CLI.

Reply:
i'm thinking that problem is not unique to the api tests.  @houshengbo may have some suggestions.